internal/user/repository.go

```diff
@@ -28,6 +29,7 @@ func (r *Repository) SaveTokenSignOn(email, token, userType string) error {
 // returns the user struct, whether the user existed already and an error
 func (r *Repository) GetOrCreateUserFromToken(token string) (User, bool, error) {
 	u := User{}
+	fmt.Println("hello world")
```

Comment:
good for now please remember to cleanup debug statement before marking PR as ready

Reply:
Removed.